node/src/components/rest_server/filters.rs

```diff
@@ -25,6 +25,9 @@ pub const STATUS_API_PATH: &str = "status";
 /// The metrics URL path.
 pub const METRICS_API_PATH: &str = "metrics";
 
+/// The OpenRPC scehma URL path.
+pub const OPEN_RPC_API_PATH: &str = "openrpc";
```

Comment:
I think we should be leaning towards emphasising that this returns a schema rather than focusing on the format of the returned schema.  I guess I'd go for something more like
```
pub const JSON_RPC_SCHEMA_API_PATH: &str = "rpc-schema";
```

Reply:
Used suggestion